packages/gui/src/components/pool/PoolOverview.tsx

```diff
@@ -3,6 +3,7 @@ import { Trans } from '@lingui/macro';
 import { useToggle } from 'react-use';
 import { Flex, UnitFormat, More, Table, mojoToChiaLocaleString, Suspender } from '@chia/core';
 import { useNavigate } from 'react-router';
+import { Alert } from '@material-ui/lab';
```

Comment:
This file has changed in main, so the imports look different. In any event, `Alert` should be imported from `@mui/material`

Reply:
Im going to catch it up to main